odoo/addons/base/models/res_partner.py

```diff
@@ -234,6 +234,7 @@ def default_get(self, default_fields):
         readonly=False, store=True,
         help='The internal user in charge of this contact.')
     vat = fields.Char(string='Tax ID', index=True, help="The Tax Identification Number. Values here will be validated based on the country format. You can use '/' to indicate that the partner is not subject to tax.")
+    vat_value = fields.Char(compute='_compute_vat_value', readonly=True)  # to display vat value avoiding potential xpath's on vat field
```

Comment:
I think the commit message should explain further why we need this non stored computed field, like explaining why there is an xpath that is breaking this. Also i'm wondering if we should not change the xpath to fix the behavior instead of adding a new field for that :thinking: 
The problem of the xpath is that he finds the field in the view and try to change it, maybe the xpath is not precise enough